src/kernels/MFEMLinearElasticityKernel.C

```diff
@@ -0,0 +1,37 @@
+#include "MFEMLinearElasticityKernel.h"
+
+registerMooseObject("PlatypusApp", MFEMLinearElasticityKernel);
+
+InputParameters
+MFEMLinearElasticityKernel::validParams()
+{
+  InputParameters params = MFEMKernel::validParams();
+  params.addClassDescription(
+      "The Cauchy stress tensor $\\sigma(u)_{ij} = \\lambda \\varepsilon_{kk} \\delta_{ij} "
+      "+ 2 \\mu \\varepsilon_{ij}$, where $\\varepsilon(u)_ij = \\frac{1}{2} \\partial_j u_i + "
+      "\\partial_i u_j$, with the weak form of "
+      "$(\\lambda \\varepsilon(\\phi_i)_{kk}, \\varepsilon(u_h)_{kk}) + 2 (\\mu "
+      "\\varepsilon(\\phi_i)_{mn}, \\varepsilon(u_h)_{mn} )$, to be added to an MFEM problem");
```

Comment:
I corrected this because I think it was wrong before, please could you double check it too?

Reply:
As above - probably preferable to keep things consistent with MFEM's documentation for this integrator, which expresses it [here](https://mfem.org/bilininteg/)

Reply:
Sometimes we write the strong form for the operator too, which looking at the example docs [here](https://mfem.org/examples/#example-2-linear-elasticity), googling how to interpret the divergence of a matrix and fiddling with the tensors looks like could be written $- \partial_k c_{kijl} u_l$. I'll leave this out since I've no idea if it'd mean anything to the user but let me know if it's useful.

Reply:
Also does this latex get rendered anywhere? I can see the the text for other classes by doing `./platypus-dbg --dump` but I'm not sure how to see rendered output.

Reply:
It will get rendered in any MooseDocs source documentation that calls 
`!syntax description /Kernels/MFEMLinearElasticityKernel` - this can probably be added to the source docs for this kernel in the other PR. 

We can elaborate more on the representations/notation in the source documentation - terms are sometimes expressed using the double inner product too.